### PR TITLE
Order Tiles

### PR DESCRIPTION
**Pull request by Jminding**

### CODE REVIEW

**Comment by Jminding** [comment]
[Order Tile](https://trello.com/c/1CAZYNUp/53-order-tile)

File: Wireframes/Wireframes/DottedSeperator/DottedSeperator.swift [hunk 1]
```diff
@@ -10,12 +10,13 @@ import SwiftUI
 struct DottedSeperator: View {
     
     @State var width: Int
-    @State var color: Color = .black
```
Comment (gavinmccabe) [inline Wireframes/Wireframes/DottedSeperator/DottedSeperator.swift:13]: Remember, even though `.black` was the default, you can still specify a custom value in the constructor.  Undo this change, customizing the color as you did below is still fine!

File: Wireframes/Wireframes/DottedSeperator/DottedSeperator.swift [hunk 2]
```diff
@@ -10,12 +10,13 @@ import SwiftUI
...
+    @State var color: Color
     
     var body: some View {
         Line()
            .stroke(style: StrokeStyle(lineWidth: 1, dash: [5]))
             .frame(width: CGFloat(width), height: 1)
+            .foregroundColor(color)
```
Comment (gavinmccabe) [inline Wireframes/Wireframes/DottedSeperator/DottedSeperator.swift:19]: Good catch, thank you!

File: Wireframes/Wireframes/DottedSeperator/DottedSeperator.swift [hunk 3]
```diff
@@ -30,6 +31,6 @@ struct Line: Shape {
 
 struct DottedSeperator_Previews: PreviewProvider {
     static var previews: some View {
-        DottedSeperator(width: 100)
+        DottedSeperator(width: 100, color: Color(#colorLiteral(red: 0, green: 0, blue: 0, alpha: 1)))
```
Comment (gavinmccabe) [inline Wireframes/Wireframes/DottedSeperator/DottedSeperator.swift:34]: This isn't necessary if the first fix is undone since there will still be a default value.

File: Wireframes/Wireframes/Order Tile/OrderTile.swift [hunk 4]
```diff
@@ -0,0 +1,75 @@
+//
```
Comment (gavinmccabe) [inline Wireframes/Wireframes/Order Tile/OrderTile.swift:1]: Does not work on smaller screens.  Should be supported all the way down to iPod touch

File: Wireframes/Wireframes/Order Tile/OrderTile.swift [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+//  Wireframes
+//
+//  Created by Jaymin Ding on 12/13/20.
+//
+
+import SwiftUI
+
+struct OrderTile: View {
+    @State var price: Double
+    @State var cleanerRating: Double
```
Comment (gavinmccabe) [inline Wireframes/Wireframes/Order Tile/OrderTile.swift:12]: Add some spacing around the state vars, see `HangerRating.swift` for an example.  In general, your file needs more spacing to make it easier to read.  Comments would greatly help too.

Comment (gavinmccabe) [inline Wireframes/Wireframes/Order Tile/OrderTile.swift:12]: Should take the following as inputs as well:

- Number of items
- Date
- Time
- Address
- Order number

File: Wireframes/Wireframes/Order Tile/StatusTile2.swift [hunk 6]
```diff
@@ -0,0 +1,58 @@
+//
```
Comment (gavinmccabe) [inline Wireframes/Wireframes/Order Tile/StatusTile2.swift:1]: This file needs to be re-named to something more appropriate.

File: Wireframes/Wireframes/Order Tile/OrderTile.swift [hunk 7]
```diff
@@ -0,0 +1,75 @@
...
+            VStack(){
+                HStack(){
+                    Spacer()
+                        .frame(width: 15)
+                    VStack(){
+                        Spacer()
+                            .frame(height: 15)
+                        Text("$\(String(format: "%.2f", price))")
+                            .foregroundColor(Color(#colorLiteral(red: 0.2666666667, green: 0.3294117647, blue: 0.5254901961, alpha: 1)))
+                            .font(.custom("HelveticaNeue", size: 17))
```
Comment (gavinmccabe) [inline Wireframes/Wireframes/Order Tile/OrderTile.swift:26]: Should be `HelveticaNeue-Medium` or `HelveticaNeue-Bold`

**Review by gavinmccabe — CHANGES_REQUESTED** [review]
Great job overall!  Here are the main things that should be fixed.  Please review all the comments for details.

- Space out your code a little more and add extra line breaks to break things into "blocks" so it's easy to read.
- This needs to work on all screen sizes down to the iPod touch
